client/src/components/SearchResult/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import sitspotImg from '../../assets/searchResult';
+import './style.css';
+
+const SearchResult = ({ resultArray }) => {
+  let counter = 1;
+  return (
+    <ul className="list">
+      {resultArray.map(item => {
+        // this class to avoid margin in the third img in every row
+        let classNames = '';
+        if (counter === 3) {
+          counter = 1;
+          classNames = 'list__item list__item__last__row';
```

Comment:
Instead of adding an extra class here, could you target the last item in the row in pure CSS instead using `nth-child()` or similar? It could help to simplify this logic